### DISCUSSION TITLE
Debian/MATE/i3: Always Light Mode instead of Dark Mode

### DISCUSSION THREAD

**Discussion by DC7IA** [hmatuschek/qdmr#213] [General]
Always getting light mode instead of dark mode with the current version from master.

Details: Debian 12, MATE Desktop, i3 window manager

Please tell me what else you need.

I'd also like to suggest a setting "Light Mode / Dark Mode / System Default" like so many other software does.

**Comment by hmatuschek** [comment]
The problem seems to be that Qt does not notice the change of widget styles. However, there is a solution by installing a dark Qt style like `kvantum-dark` or any other. Then, you can pass the style via command line option to qdmr. That is
```
qdmr -style kvantum-dark
```
If you always run in dark mode, consider also setting the environment variable `QT_STYLE_OVERRIDE`, then you do not need to pass it to qdmr on every startup. 

There is little I can do do set the theme from inside qdmr. I would need to ship my own widget theme alongside with qdmr. This will then likely not match the desktop theme. However, I've already implemented the detection of a dark theme in qdmr to fix the icons. 

![Bildschirmfoto von 2022-03-18 09-23-58](https://user-images.githubusercontent.com/282387/158963679-558c1b92-7b56-4ba7-95cc-b912aa88b934.png)
